app/views/layouts/_navigation.html.erb

```diff
@@ -0,0 +1,26 @@
+
+
+<nav class="bg-white border-gray-200 dark:bg-gray-900">
```

Comment:
I don't recall exactly how to do it, but how would you feel about having the navbar always visible even when scrolling? If that were to be done, I think the navbar would need to have a different background color than the main content otherwise it might look weird.

Reply:
I'm ok with that. I believe it's the fixed position in css.